c10/xpu/XPUFunctions.cpp

```diff
@@ -154,6 +160,16 @@ DeviceIndex get_device_idx_from_pointer(void* ptr) {
       std::distance(gDevicePool.devices.begin(), it));
 }
 
+DeviceIndex get_device_idx_from_uuid(const uint8_t device_uuid[16]) {
```

Comment:
why we do not use std::array<unsigned char, 16> for uuid definition here?

Reply:
because the pti gives the **uint8_t device_uuid[16]**